openedx/adg/common/helpers.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Why have we created this helper function in the `common` package? instead of in the `admin_dashboard/performance/helpers.py` module?

Reply:
In that case it should be moved to `openedx/adg/helpers.py`. Moving it inside `common` means it is only being used within the `common` package, which is not the case.

Reply:
Isn't it like common can be used in lms and cms and now admin_dashboard?

Reply:
Because we haven't configured our tests workflow for this as well. Tests run for adg/lms adg/cms and adg/common.